### DISCUSSION TITLE
2020/09/20/github-codespaces-and-terraform/

### DISCUSSION THREAD

**Discussion by giscus** [russmckendrick/blog#14] [Comment]
# 2020/09/20/github-codespaces-and-terraform/

I got my invite for GitHub Codespaces the other day and I thought I would have a quick play with it as having an an online IDE sounds like it could come in handy.
If you know me — you will know that I am not a developer, but I do a lot of work in Visual Studio Code as that is where I tend to develop work on a lot of the infrastructure as code projects using Terraform and Ansible, as well as Docker projects.

https://www.mediaglasses.blog/2020/09/20/github-codespaces-and-terraform/

**Comment by ttbakiatwoam** [comment]
Your script is incorrect, you have /bin/sh when you wrote a BASH script. Sh will not handle the array you're defining. If you try and run your script using "sh" you will get an error; post-install.sh: 9: Syntax error: "(" unexpected (expecting "}"). However, since this script is ran by BASH at startup you probably never noticed.

The script secussfully executes with bash post-install.sh at the command line where sh post-install.sh fails. I would update the #! line to read #!/bin/bash. Yes it's a little change but it will save someone else time.
